packages/web/src/components/transaction-details-modal/components/Block.tsx

```diff
@@ -0,0 +1,26 @@
+import { ComponentPropsWithoutRef, ReactNode } from 'react'
+
+import cn from 'classnames'
+
+import styles from './Block.module.css'
+
+export const Block = ({
+  header,
+  children,
+  className,
+  ...divProps
+}: {
+  header: ReactNode
+  children: ReactNode
+} & ComponentPropsWithoutRef<'div'>) => {
+  return (
+    <div className={cn(styles.block, className)} {...divProps}>
+      <div className={styles.blockHeader}>{header}</div>
+      <div className={styles.blockContent}>{children}</div>
+    </div>
+  )
+}
+
+export const BlockContainer = ({ children }: { children: ReactNode }) => {
```

Comment:
do we want these to be lists? if so container could be `<ul />` (might want to rename to BlockList), and blocks would render `<li>`

Reply:
🧠